### ISSUE TITLE
[Discussion]: Thread for 'Envisioning Communities: A Participatory Approach Towards AI for Social Good' paper

### ISSUE THREAD

**Issue by harisood** [alan-turing-institute/research-application-management#54]
### Summary

Link to paper: https://arxiv.org/pdf/2105.01774.pdf

### Detail

A thread to share thoughts on the above paper, and if/how we want to use it to shape Project RAM

### Intended Output

Agreement on how this will shape Project RAM

**Comment by dingaaling** [comment]
**Open questions from 6 Oct (with @f-rower & @S-Laher):**
- Capabilities approach is theoretically interesting, but how do you operationalise?
- How do we improve the user experience of better data and AI practices (e.g. opt in to data cookies)?
- Addressing feeling of learned hopelessness vs. empowerment?
- Who needs to be on the team (or as a partner) to ^ address these questions beyond academics (e.g. designers, engagement experts)?
- @S-Laher mentioned that generally outputs focus on academic paper or white papers which is primarily used as PR or "putting marker in the ground" but background in teaching could bring unqiue expertise to help differentiate to have the most impact
- @f-rower mentions how they considered changing the PitchFest prompt this year to orient more around accessible communication, but instead the team has decided to focus on diversifying the judging panel; also brings up the point on how AI has become a public topic so many more aware of capabilities
- @aranas working on a CDT training about how to write abstract in layman's terms - could be a training session for PitchFest participants
- @f-rower brings up the point about how do you measure expertise in something like "AI"? @S-Laher points out that titles don't always convey expertise especially when "AI" is something so broad. Perhaps better to focus on their background/experience as a way to identify what their expertise is and points out the Wired compilation of "5 levels of explanation" to show the range

**Comment by harisood** [comment]
This is cool! I wonder if we could write a response to the authors and reach out to them? Wdwt?

**Comment by dingaaling** [comment]
I have a lot of new thoughts on this topic coming out of the Connected by Data Design Lab on Public Deliberation on AI! They organised a [People's Panel on AI](https://connectedbydata.org/projects/2023-peoples-panel-on-ai) and have also partnered with an organisation that put together the [Global Citizen's Assembly on Climate](https://globalassembly.org/about-2). Also related is the [Collective Intelligence Project](https://cip.org/).

My point for now is that the Capabilities approach in this paper is one framework of many we could consider, re: participatory methods, and I'd argue less direct to operationalise compared to the tested methods that orgs in our network like Connected by Data, Ada Lovelace, and Iswe.

It's not entirely clear to me what an opportunity for a next step here is for us besides discussion... but perhaps:
- @aranas and I could explore ways to embed some public deliberation methods into an activity for the DyME workshop in Feb 2024 if that ends up going forward
- those of who are interested can engage further with Connected by Data who are interested in funding/running more public deliberations next year
- @aldenc and other E&S affiliated RAMs could explore through the WP1 community activities to test some participatory methods

**Comment by aldenc** [comment]
add to strategy epic
